### DISCUSSION TITLE
Where are all the announcement posts going? A note on some current tidying in the Announcements Section 🧹✨

### DISCUSSION THREAD

**Discussion by novium2** [community/community#137981] [Announcements]
Wondering where an old announcement has gone? Looking for a great link you saw in an old post here? They are still all available, just switch your view to "closed" to find old announcements. We are doing a bit of tidying in preparation of Universe 2024, and that means closing some of the oldest announcements here to ensure that when you visit the Announcements category, you only see the latest and most recent news.

**Comment by MasterInQuestion** [comment]
[ novium2 @ CE 2024-09-06 23:22:59 UTC:
[https://github.com/orgs/community/discussions/137981#discussion-7148056](https://github.com/orgs/community/discussions/137981#discussion-7148056)
    Wondering where an old announcement has gone?
    Looking for a great link you saw in an old post here?
    They are still all available, just switch your view to "closed" to find old announcements.

    We are doing a bit of tidying in preparation of Universe 2024.
    And that means closing some of the oldest announcements here, to ensure that:
    When you visit the Announcements category, you only see the latest and most recent news. ]
.
    Related:
    https://github.com/orgs/community/discussions/138192#discussioncomment-10595394

**Comment by MohammadHonarkar** [comment]
[Mohammad “Mo” Honarkar](https://www.tidewaternews.com/business/mohammad-honarkar-laguna-beach-entrepreneur/) is a prominent entrepreneur and real estate developer based in Orange County, California. Renowned for his influential role in the region's real estate landscape, Honarkar has made significant strides, particularly in the [Laguna Beach area](https://www.lagunabeachindy.com/once-strictly-private-museum-style-space-will-open-to-public-with-daring-art-exhibition/). His work is characterized by a commitment to revitalizing and enhancing the communities he is involved in.
--
 
With a deep understanding of the local market and a vision for sustainable development, [Honarkar ](https://network.axial.net/company/mohammad-honarkar)has led numerous successful projects that blend innovation with community-oriented values. His ventures often focus on creating vibrant, dynamic spaces that contribute to the cultural and [economic growth](https://www.thehonarkarfoundation.org/) of the area.
 
[Honarkar’s achievements](https://about.me/MohammadHonarkar) are a testament to his expertise and dedication to his field, earning him a reputation as a key player in the real estate development sector in [Southern California](https://www.thehonarkarfoundation.org/mission/). His contributions extend beyond development, reflecting a broader impact on the region’s growth and aesthetic evolution.
